proto/process/agent.proto

```diff
@@ -1591,3 +1591,106 @@ message VPACondition {
 	string Reason = 4;
 	string Message = 5;
 }
+
+// https://github.com/kubernetes/api/blob/master/autoscaling/v2/types.go
+message HorizontalPodAutoscaler {
+	Metadata metadata = 1;
+	HorizontalPodAutoscalerSpec spec = 2;
+	HorizontalPodAutoscalerStatus status = 3;
+	bytes yaml = 4 [deprecated = true];
+	repeated string tags = 5;
+	repeated HorizontalPodAutoscalerCondition conditions = 6;
+}
+
+message HorizontalPodAutoscalerSpec {
+	HorizontalPodAutoscalerTarget target = 1;
```

Comment:
seems that there's inconsistency in the indentation (tabs above and space below)